.gitignore

```diff
@@ -0,0 +1,55 @@
+# Ignore bundler config
```

Comment:
Thank you for the lesson and walkthrough on .gitignore, it was a huge benefit to the group's understanding.